### DISCUSSION TITLE
How to avoid page being pushed to the outside of the screen when the keyboard pops up?

### DISCUSSION THREAD

**Discussion by trojanyao** [expo/expo#30119] [Archived: Questions]
Hi there, I have a page in Expo which layout is like the screenshot below.

<img width="50%" src="https://github.com/expo/expo/assets/22811809/5cca3df7-8d9f-45bd-a64c-140e5abc9e88">

On the Web (iOS Safari), every time when the keyboard pops up, the whole page was pushed up out of screen (below).

<img width="50%" src="https://github.com/expo/expo/assets/22811809/acc24924-e25e-40f1-82ed-e0b583addf5f">

Expected behavior: the Header still fixed at the top of page, and the height of Chat box can be compressed.

Anyone know how to resolve this? I tried `KeyboardAvoidingView`, but it looks like not for this scenario.

My code is as follows.

<img width="1013" alt="Xnip2024-07-02_14-38-55" src="https://github.com/expo/expo/assets/22811809/ed0bfd50-ddc0-4ab1-b100-dcf35291fed8">

**Comment by tomsjansons** [comment]
Did you ever solve this? I'm seeing something similar
